Source/Urho3D/Engine/Engine.cpp

```diff
@@ -1054,14 +1054,14 @@ void Engine::PopulateDefaultParameters()
     engineParameters_->DefineVariable(EP_TRIPLE_BUFFER, false);
     engineParameters_->DefineVariable(EP_VALIDATE_SHADERS, false);
     engineParameters_->DefineVariable(EP_VSYNC, false).Overridable();
-    engineParameters_->DefineVariable(EP_WINDOW_HEIGHT, 0); //.Overridable();
+    engineParameters_->DefineVariable(EP_WINDOW_HEIGHT, 0).Overridable();
```

Comment:
Please don't put multiple unrelated changes in a single PR. It is much easier to evaluate each change on it's own.

Reply:
Yeah, those were disabled because they in fact don't work on Retina displays.
It is fixable but I decided to postpone fix until we get Diligent working.